source/_components/homematicip_cloud.markdown

```diff
@@ -23,18 +23,18 @@ To set up the component:
 generate_auth_token.py
 ```
 
-- ** add the information to your `configuration.yaml` file:
+- **add the information to your `configuration.yaml` file**:
```

Comment:
Not sure why this piece of text needs to be printed so strongly.
I suggest removing the bold markers.